CRM/Utils/Check/Component/Source.php

```diff
@@ -17,90 +17,28 @@
 class CRM_Utils_Check_Component_Source extends CRM_Utils_Check_Component {
 
   public function getRemovedFiles() {
-    $files[] = '[civicrm.packages]/Auth/SASL';
-    $files[] = '[civicrm.packages]/Auth/SASL.php';
-    $files[] = '[civicrm.packages]/Net/SMTP.php';
-    $files[] = '[civicrm.packages]/Net/Socket.php';
-    $files[] = '[civicrm.packages]/_ORIGINAL_/Net/SMTP.php';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/Readme.md';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/license.txt';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/media/css/jquery.dataTables.css';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/media/css/jquery.dataTables.min.css';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/media/css/jquery.dataTables_themeroller.css';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/media/js/jquery.dataTables.js';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/media/js/jquery.dataTables.min.js';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/media/js/jquery.js';
```

Comment:
The problem being solved is this: _Merely unzipping the new code release doesn't delete files, esp on Joomla._ (*It's true anywhere, but Joomla doesn't have alternative workflows to get around the issue.*)

AFAICS, the same problem applies equally to `civicrm-core` and `civicrm-packages`.

Reply:
@totten this PR does include `civicrm-packages`.